schema.development.kf

```diff
@@ -669,31 +710,15 @@ procedure get_credential_shared ($id uuid) public view returns table (
         error('the credential does not exist');
     }
 
-    $ext_ag_granted bool := false;
-    $ext_ag_grant_valid bool := false;
     $int_ag_granted bool := false;
 
-    for $ext_ag_row in SELECT status FROM external_access_grants WHERE data_id = $id::TEXT AND ag_grantee = @caller COLLATE NOCASE {
-        $ext_ag_granted := true;
-        if $ext_ag_row.status == 'valid' {
-            $ext_ag_grant_valid := true;
-            break;
-        }
-    }
-
-    for $int_ag_row in SELECT 1 FROM internal_access_grants WHERE data_id = $id AND ag_grantee_wallet_identifier = @caller COLLATE NOCASE {
+    for $int_ag_row in SELECT 1 FROM access_grants WHERE data_id = $id AND ag_grantee_wallet_identifier = @caller COLLATE NOCASE {
```

Comment:
We can `break` after the assignment, no?

Reply:
Good call. Added.